PROPOSAL.md

```diff
@@ -0,0 +1,181 @@
+The Ledger & Tokenization working group plans to work on an _optional_ core extension specifying the interface for fetching transactions.
+
+
+### Metadata access
+
+ICRC-1 defines a [generic metadata mechanism](https://github.com/dfinity/ICRC-1#metadata) allowing ledger implementations to define application-specific attributes, such as token logos and URLs (see the [icrc1_metadata](https://github.com/dfinity/ICRC-1#icrc1_metadata-) method).
+
+
+### Modularity
+
+ICRC-1 is the _base_ standard designed for extension with follow-up specifications as new flows become practical.
+Many efficient flows require unique features of the IC platform, such as [Threshold ECDSA signatures](https://medium.com/dfinity/threshold-ecdsa-the-key-ingredient-behind-the-internet-computers-bitcoin-and-ethereum-cf22649b98a1), to become stable and well-understood.
+The standard defines the [icrc1_supported_standards](https://github.com/dfinity/ICRC-1#icrc1_supported_standards) method listing all standards the token ledger supports.
+This method will allow applications such as wallets and Rosetta API implementations to discover and take advantage of additional features in the future.
+
+The Ledger & Tokenization working group will continue working on a set of _core_ extensions recommended for all ledger implementations.
+Furthermore, anyone can define new extensions and include them into the supported standards list.
+For example, existing [DIP20](https://github.com/Psychedelic/DIP20)-compatible ledgers can include DIP-20 into the supported standards list, allowing compatible clients to discover and access the corresponding API.
+
+
+### Transaction deduplication semantics
+
+ICRC-1 defines a [protocol for transaction deduplication](https://github.com/dfinity/ICRC-1#transaction-deduplication-) to help clients avoid accidental double spending in the presence of temporary networking issues without the need to implement complex recovery mechanisms.
+
+
+## Interoperability of ICRC-1 and the ICP ledger
+
+The DFINITY Foundation plans to implement the ICRC-1 interface in the ICP ledger canister smart contract.
+Though ICRC-1 is compatible with the ICP ledger design, transferring funds to 28-byte ICP account identifiers will not be possible through the ICRC-1 interface.
+You will have to use the existing ICP ledger interface for such transfers.
+
+
+## Conclusion
+
+The [Ledger & Tokenization working group](https://forum.dfinity.org/t/announcing-token-standard-as-topic-of-the-first-meeting-of-the-ledger-tokenization-working-group/11925/1) proposes to adopt the [ICRC-1](https://github.com/dfinity/ICRC-1) token standard as a common interface for transferring fungible tokens on the Internet Computer.
+The standard does not seek to replace existing token ledgers but rather to serve as a base interface for accessing them.
+The working group agrees to implement, integrate, and promote the standard and work on its core extensions.
```

Comment:
```suggestion
The working group agrees to implement, integrate, and promote the standard and work on its optional extensions.
```
Make it clear that an extension is always optional and is not needed to implement the ICRC-1 standard.